src/routers/validateRouteConfigMap.js

```diff
@@ -55,6 +55,14 @@ function validateRouteConfigMap(routeConfigs: NavigationRouteConfigMap) {
           '}'
       );
     }
+
+    if (routeConfig.path) {
+      invariant(
+        routeConfig.path[0] !== '/',
+        `Invalid path '${routeConfig.path}' for route '${routeName}'. ` +
+          `Path should not start with the slash symbol`
```

Comment:
I think it would be best to be explicit with the error message and say `Path should not start with the "/" character.` "Slash" could be slightly ambiguous since the slash could be a forward or back slash, yet only one throws an error.